### DISCUSSION TITLE
実際にビルドガイドを通して引っかかったポイント

### DISCUSSION THREAD

**Discussion by 84monta** [Yowkees/keyball#92] [General]
非常に丁寧なビルドガイドでほとんど困らなかったのですが、いくつか気になった点がありましたので共有します。
*不慣れな人間がやると、この辺に戸惑うということで、アップデートのご参考に

1) OLEDにもピンが付属しており、どちらを使うべきかまよった。
別の4ピンを使っていると思いますが、OLEDにも袋の中に標準でついておりどちらを使うべきか迷った。
全体を見ると短いほうの4ピンが他に使用しているポイントがなかったので（プラスチックの部分が）短いものを使用した。

2) 読取りIC用レンズの小さな足に関する注意
細い足が出ているが、生産上のバリか何か(もしくは折れることで、返しのようになって抜けなくなる）かと思っていたので、穴とずれていることに気づかず押し付けて折れた。
ビルドガイドではさらっと取り付けることが書いてあるので、足と奥にある穴を合わせて優しく差し込むことを説明記載いただければ、悲劇を防げるかと思います。

3) ロープロを組み合わせる場合、4㎜のスペーサーが必要であることを最初に書く
親指にロープロを使用する場合にユーザー側でそろえるべきものを現状の1章にまとめて記載いただけるとうっかりな人も気づきやすいと思います。
（7-2まで気づきませんでした）

**Comment by Yowkees** [comment]
84monta様

ご指摘いただきました、Keyball46のビルドガイド該当箇所を修正してみました。
貴重なご意見ありがとうございました。
